src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs

```diff
@@ -0,0 +1,2156 @@
+//-----------------------------------------------------------------------
+// <copyright file="SimpleJson.cs" company="The Outercurve Foundation">
+//    Copyright (c) 2011, The Outercurve Foundation.
+//
+//    Licensed under the MIT License (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//      http://www.opensource.org/licenses/mit-license.php
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+// </copyright>
+// <author>Nathan Totten (ntotten.com), Jim Zimmerman (jimzimmerman.com) and Prabir Shrestha (prabir.me)</author>
+// <website>https://github.com/facebook-csharp-sdk/simple-json</website>
+//-----------------------------------------------------------------------
+
+// VERSION:
+
+// NOTE: uncomment the following line to make SimpleJson class internal.
+#define SIMPLE_JSON_INTERNAL
```

Comment:
Should we make this all internal? Is there a reason we should make this available outside of the transport?

Reply:
It is internal because the define is uncommented

Reply:
Are you sure, am I reading this wrong?

https://github.com/Particular/NServiceBus.AmazonSQS/blob/3271c36541bd7c4d856f0e39169f5e376f029047/src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs#L497-L502

Reply:
Nevermind. I am reading it wrong.